mediorum/server/serve_legacy.go

```diff
@@ -105,7 +105,13 @@ func (ss *MediorumServer) findHostsWithCid(ctx context.Context, cid string) ([]s
 
 func (ss *MediorumServer) isCidBlacklisted(ctx context.Context, cid string) bool {
 	blacklisted := false
-	sql := `select count(*) = 1 from "ContentBlacklists" where "value" = $1 and "type" = 'CID'`
```

Comment:
can we drop this table?

Reply:
I want to keep it to be safe in case we need to compare or debug existing /blacklist routes, but after successful rollout I can remove this table, the /blacklist routes, BlacklistManager, etc...